src/common/components/select-repository-list/index.js

```diff
@@ -125,6 +125,25 @@ export class SelectRepositoryListComponent extends Component {
     return renderedRepos;
   }
 
+  renderRepoAmount() {
+    const { ids, isFetching } = this.props.repositories;
+    const { selectedRepositories } = this.props;
+
+    if (isFetching && ids.length === 0) {
+      return (
+        <span>Calculating your total repositories&hellip;</span>
+      );
+    }
+
+    return (
+      <div>
+        <strong>
+          { selectedRepositories.length } selected out of { ids.length } repositories
```

Comment:
[Spec says](https://docs.google.com/document/d/1koTwI3cRr2S2vj5zHKKbR7aWxLRfShDVq2gHQ29cDyU/edit#) it should be "{number} selected of {total} repos"